### DISCUSSION TITLE
Error when using custom validator

### DISCUSSION THREAD

**Discussion by oni-on1003** [opencrvs/opencrvs-core#7787] [Technical Q&A] [answered]
I've got an error when using the custom validator `isNumberLessThan21`
`Error: Cannot find a validator with a name isNumberLessThan21.
    This is a configuration error in your country specific resource package's form field definitions.`

Usage:
`export function testCustomValidator(...):SerializedFormField{.........,validator: [
      {
        operation: 'isNumberLessThan21'
      }
    ]
}`

**Comment by naftis** [comment]
@oni-on1003 Are you still encountering this issue?

**Reply by oni-on1003** [reply, marked as answer]
No more errors in v1.6.1
